### DISCUSSION TITLE
API para retornar grafo de ligações

### DISCUSSION THREAD

**Discussion by andersonvass** [rictom/rede-cnpj#30] [General]
Bom dia,

Há no projeto alguma forma de chamar uma API que retorna o grafo das ligações em imagem ou SVG, passando o CNPJ ou CPF e o nível de profundidade do grafo?

No aguardo

Até mais

**Comment by rictom** [comment]
Essa é uma idéia interessante. Atualmente não tem como fazer isso, já que o gráfico é gerado no navegador com javascript. Teria que refazer a geração de gráfico somente do lado do servidor, talvez dê pra fazer uma visualização mais simples usando uma biblioteca como o graphviz.
